TABTestKit/Classes/TABTestCase.swift

```diff
@@ -9,6 +9,10 @@
 //
 
 import XCTest
+#if !COCOAPODS
```

Comment:
Only import Biometrics and BDD if not using Cocoapods.

Reply:
If you don't import Biometrics then surely anything that uses Biometrics in this file would fail to  compile?